docs/reference/sparse-api/helpers.rst

```diff
@@ -8,16 +8,16 @@
 Sparse matrix helper functions
 *******************************
 
-These are helper functions that control aspects of the hipSOLVER library. They are divided
+These helper functions control aspects of the hipSOLVER library. They are divided
 into the following categories:
 
-* :ref:`sparse_initialize` functions. Used to initialize and cleanup the library handle.
-* :ref:`sparse_stream` functions. Provide functionality to manipulate streams.
+* :ref:`sparse_initialize` functions initialize and cleanup the library handle.
+* :ref:`sparse_stream` functions provide functionality to manipulate streams.
```

Comment:
No colons? Ideally these would be consistent with descriptions in dense-api/helpers.rst